ethicml/algorithms/inprocess/in_algorithm.py

```diff
@@ -37,7 +46,9 @@ def fit(self: _I, train: DataTuple) -> _I:
         """
 
     @property
-    def hyperparameters(self) -> Dict[str, Union[str, int, float]]:
+    def hyperparameters(
+        self,
+    ) -> Dict[str, Union[str, int, float, KernelType, ActivationType, FairnessType]]:
```

Comment:
This is a very specific return type for an abstract base class...

Reply:
yeah, this should just be `str -> Any` I guess